modules/nf-core/bcftools/norm/main.nf

```diff
@@ -12,19 +12,24 @@ process BCFTOOLS_NORM {
     path(fasta)
 
     output:
-    tuple val(meta), path("*.gz") , emit: vcf
-    path "versions.yml"           , emit: versions
+    tuple val(meta), path("*.{vcf,vcf.gz,bcf,bcf.gz}")  , emit: vcf
```

Comment:
in line with other modules, I would suggest to divide between vcf and bcf  https://github.com/nf-core/modules/blob/master/modules/nf-core/samtools/view/main.nf

Reply:
Thanks for the suggestion! Would you also divide between the compressed and uncompressed versions (e.g. vcf and .vcf.gz)?

Reply:
yes, it would ensure people really get out what they expect.